.gitignore

```diff
@@ -1,15 +1,26 @@
-# Created by https://www.toptal.com/developers/gitignore/api/python
-# Edit at https://www.toptal.com/developers/gitignore?templates=python
+## Project specific gitignore entries go up front to make them easy to find:
 
-### Python ###
+# Ignore JSON Spec files that belong to ConnectWise
+ConnectWiseManage.json
+/All.json
+/automate_json/
+
+# Are these still needed?
```

Comment:
No longer needed - but already fixed, so I won't bother you with updating this :)